odl/contrib/fom/supervised.py

```diff
@@ -460,7 +481,7 @@ def false_structures_mask(foreground, smoothness_factor=None):
     unique = np.unique(foreground)
     if not np.array_equiv(unique, [0., 1.]):
         raise ValueError('`foreground` is not a binary mask or has '
-                          'no true values {!r}'.format(unique))
+                         'no true values {!r}'.format(unique))
```

Comment:
Is it ``truth values``?

Reply:
I think it is `true values`. The mask is suppose to indicate which parts of the image that should be de-emphasized (the foreground), and this error is returned if the values in `foreground` does not take exactly the values 0 and 1, i.e., nothing else than 0 and 1 and at least one pixel with each value. But that means that the error message is wrong anyway, because a mask of only ones (so all values are `true`) would also return this error.